content/how-to/cloud-servers/multifactor-authentication-from-the-cloud-control-panel/index.md

```diff
@@ -191,28 +211,23 @@ account.
 
 1. Log in to the [Rackspace Cloud Control Panel](https://login.rackspace.com).
 
-2. In the top navigation bar, click **Select a Product > Rackspace Cloud**.
+2. In the upper-right, click **Account** > **Account Settings**.
 
-3. In the upper-right, click **Account** > **Account Settings**. Under
-   **Rackspace Account Settings**, click **Change** next to **Multi-Factor
-   Authentication**.
+   <img width="311" alt="3 2" src="https://user-images.githubusercontent.com/96761530/161067080-ed2759d3-b497-4b4d-a632-2726dbf023c9.png">
 
-4. In the **Security Settings** section, select **Enable** for **Multi-factor
-   authentication**.
+3.  Under **Rackspace Account Settings**, click **Change** next to Multi-Factor
+   Authentication.
+   
+   <img width="485" alt="3 3" src="https://user-images.githubusercontent.com/96761530/161067197-9a33a96b-0e40-4378-a2fa-c3bd3973ccf0.png">
 
-5. On the **Account-wide two-factor authentication** form, click the selection
-   to set the policy for account users. Then, click **Save Setting** to apply
-   the change.
+4. In the Pop-up window, click **Require Multi-Factor Account-Wide**.
 
-   If you update the setting to **required**, users who do not have MFA
-   configured must add it the next time they log in. The system
-   logs out current users who have not set up MFA after
-   displaying an error message similar to the following example:
+   <img width="384" alt="MFA ADMIN" src="https://user-images.githubusercontent.com/96761530/161553015-771ac56b-f0d0-402a-b182-fcb65bb22ba6.png">
 
-   {{<image alt="Force user to log out when multi-factor authentication requirement is not met" src="mfa-force-logout.png" title="Force user to log out when multi-factor authentication requirement is not met">}}
 
-   When these users log back in, the system guides them through the
-   MFA setup.
+   If you update the setting to **require Multi-Factor Account-Wide**, any users currently logged into the Rackspace Control Panel without Multi-Factor   Authentiocation enabled will be logged out and asked to set up multi-factor upon re-entry.
```

Comment:
```suggestion
   If you update the setting to **require Multi-Factor Account-Wide**, any users currently logged into the Rackspace Control Panel without Multi-Factor Authentiocation enabled will be logged out and asked to set up multi-factor upon re-entry.
```
I'm just cleaning up an added space you had above. You can commit my suggestion if it is correct.